pkg/providers/tinkerbell/assert_test.go

```diff
@@ -229,6 +229,41 @@ func TestAssertTinkerbellIPAndControlPlaneIPNotSame_SameFails(t *testing.T) {
 	g.Expect(tinkerbell.AssertTinkerbellIPAndControlPlaneIPNotSame(clusterSpec)).ToNot(gomega.Succeed())
 }
 
+func TestAssertPortsNotInUse_Succeeds(t *testing.T) {
+	g := gomega.NewWithT(t)
+	ctrl := gomock.NewController(t)
+
+	netClient := mocks.NewMockNetClient(ctrl)
+	netClient.EXPECT().
+		DialTimeout(gomock.Any(), gomock.Any(), gomock.Any()).
```

Comment:
Only use `gomock.Any()` if you can't determine the value the mock client is expecting